.github/workflows/publish-release.yml

```
@@ -699,11 +699,11 @@ jobs:
       - name: Set up JDK 21
         uses: actions/setup-java@v2
         with:
-          distribution: 'temurin'
-          java-version: '21.0.3'
+          distribution: "temurin"
+          java-version: "21.0.3"
       - uses: actions/setup-dotnet@v1
```

actionlint (1.7.12)

[error] the runner of "actions/setup-dotnet@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)